docs-2.0/7.data-security/1.authentication/3.role-list.md

```diff
@@ -54,7 +54,7 @@ NebulaGraph 内置了多种角色，说明如下：
 
   - 对权限内的图空间可以自定义 Tag 和 Edge Type 的数据的**读写**权限。
 
-!!! enterpriseonly
+!!! 企业版专有
```

Comment:
Here is the special format of our documents, please keep it here`!!! enterpriseonly`

Reply:
My concern is the usage of this term 'enterpriseonly' in the Chinese version of the document. And, 'enterpriseonly' shall be 'enterprise-only'. Maybe we'd better update the special format?

Reply:
This is only a macro